### PR TITLE
feature #202: authoritative on activate_apis

### PR DESCRIPTION
**Pull request by alexkonkin**

### CODE REVIEW

File: modules/core_project_factory/main.tf [hunk 1]
```diff
@@ -123,7 +123,7 @@ resource "google_resource_manager_lien" "lien" {
   APIs configuration
  *****************************************/
 resource "google_project_service" "project_services" {
-  count = "${length(var.activate_apis)}"
+  count = "${ var.apis_authority == "true" ? 0 : length(var.activate_apis)}"
```
Comment (aaron-lane) [inline modules/core_project_factory/main.tf]: This expression can be simplified:

```suggestion
  count = "${var.apis_authority ? 0 : length(var.activate_apis)}"
```

File: modules/core_project_factory/main.tf [hunk 2]
```diff
@@ -134,6 +134,15 @@ resource "google_project_service" "project_services" {
   depends_on = ["google_project.main"]
 }
 
+resource "google_project_services" "project_services_authority" {
+  count = "${ var.apis_authority == "true" ? 1 : 0 }"
```
Comment (aaron-lane) [inline modules/core_project_factory/main.tf]: This expression can be simplified:

```suggestion
  count = "${var.apis_authority ? 1 : 0}"
```

File: variables.tf [hunk 3]
```diff
@@ -66,6 +66,11 @@ variable "sa_role" {
   default     = ""
 }
 
+variable "apis_authority" {
+  description = "Defines which type of project_service to use: project_services/project_services_authority"
```
Comment (aaron-lane) [inline variables.tf]: This description should be clarified:

```suggestion
  description = "Toggles authoritative management of project services."
```

File: modules/core_project_factory/variables.tf [hunk 4]
```diff
@@ -71,6 +71,11 @@ variable "sa_role" {
   default     = ""
 }
 
+variable "apis_authority" {
+  description = "Defines which type of project_service to use: project_services/project_services_authority"
```
Comment (aaron-lane) [inline modules/core_project_factory/variables.tf]: This description should be clarified:

```suggestion
  description = "Toggles authoritative management of project services."
```

**Comment by aaron-lane** [comment]
Fixes #213

File: modules/core_project_factory/main.tf (hunk 1 above)
Comment (alexkonkin) [reply modules/core_project_factory/main.tf]: Thank you for your review. Done.

File: modules/core_project_factory/main.tf (hunk 2 above)
Comment (alexkonkin) [reply modules/core_project_factory/main.tf]: Thank you for your review. I will keep this in mind.

**Comment by aaron-lane** [comment]
@alexkonkin It appears that the branch history has been confused; there are commits included which have already been merged to the master branch. Can you rebase this branch against terraform-google-modules:master?

**Comment by aaron-lane** [comment]
Fixes #202.

**Comment by alexkonkin** [comment]
> 
> 
> Please add an entry to the Unreleased section of the CHANGELOG and regenerate the documentation once the requested changes are applied.

Done.

**Pull request merged**